### DISCUSSION TITLE
Caddyserver error

### DISCUSSION THREAD

**Discussion by Afolic** [projectdiscovery/nuclei#2128] [Q&A] [answered]
Upon trying to install nuclei, I keep getting this error message "go/pkg/mod/github.com/caddyserver/certmagic@v0.16.1/account.go:28:2: package io/fs is not in GOROOT (/usr/lib/go/src/io/fs)"

**Comment by forgedhallpass** [comment, marked as answer]
Are you trying to build or install nuclei? Are you using the correct command and version of GO as described in https://github.com/projectdiscovery/nuclei#install-nuclei?

**Reply by Afolic** [reply]
Yes I am, "go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest"

**Reply by forgedhallpass** [reply]
Could you share a screenshot with the command, including `go version`?
